### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Kacperch1**
add-shop-simulation

### CODE REVIEW

File: cart.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Cart:
+    def __init__(self):
+        self.products: list = []
+
```
Comment (1WojciechNowak) [inline cart.py]: Lines since 4th needs to be deleted

File: main.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+from product import Product
+if __name__ == "__main__":
+    cart = Cart()
+    apple = Product("apple", 0.5)
+    bread = Product("bread", 6)
+    water = Product("water", 2)
+    orange = Product("orange", 1)
+## jak wygląda sklep?roduct
+## 1.wchodzę witam się 2.mówię sprzedawcy co potrzebuję 3. gdy mam wszystko płace 4. pakuję to do siatki żegnam się i ide
+    print(input("say hello: "))
```
Comment (1WojciechNowak) [inline main.py:11]: Do not repeat yourself. What u could do instead repeating the code?

File: product.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Product:
+    def __init__(self, name: str, price: int):
+        self.name = name
+        self.price = price
+
```
Comment (1WojciechNowak) [inline product.py]: Lines!

**Review by 1WojciechNowak — CHANGES_REQUESTED** [review]

File: cart.py (hunk 1 above)
Comment (Kacperch1) [reply cart.py]: done

File: product.py (hunk 3 above)
Comment (Kacperch1) [reply product.py]: done

File: main.py (hunk 2 above)
Comment (Kacperch1) [reply main.py:11]: done

**Review by 1WojciechNowak — APPROVED** [review]

**Pull request merged**